src/main/python/camdkit/model.py

```diff
@@ -117,23 +733,23 @@ class FStop(StrictlyPositiveIntegerParameter):
   canonical_name = "fStop"
   sampling = Sampling.REGULAR
   units = "0.001 unit"
+  section = "lens"
 
-class FocalLength(StrictlyPositiveIntegerParameter):
-  """Nominal focal length of the lens. The number printed on the side of a prime
-  lens, e.g. 50 mm, and undefined in the case of a zoom lens."""
+class FocalLength(NonNegativeRealParameter):
+  """Focal length of the lens."""
```

Comment:
The long description text should go back in since it was carefully crafted.

Reply:
Added a separate field NominalFocalLength for this